src/cd.js

```diff
@@ -8,12 +8,21 @@ function _cd(options, dir) {
   if (!dir)
     common.error('directory not specified');
 
+  if (dir === '-') {
+    var previous_dir = true;
+    if (!common.state.previousDir)
+      common.error('OLDPWD not set');
```

Comment:
Sure thing. I just copied the error message from bash. I agree, it probably should be more descriptive.